### ISSUE TITLE
New-UDGrid: "MUI X Expired license key"

### ISSUE THREAD

**Issue by Viajaz** [ironmansoftware/powershell-universal#2596]
### Version

4.0.9

### Severity

Low

### Steps to Reproduce

In some situations, when UDGrid has a filter set, it shows in grey a message in the center of the grid saying "MUI X Expired license key"

### Expected behavior

```console
Not to show "MUI X Expired license key"
```

### Actual behavior

```console
"MUI X Expired license key"
```

**Comment by Viajaz** [comment]
@adamdriscoll 
This also shows up in large tables, I have lots of user inquiries about this.

**Comment by Viajaz** [comment]
@adamdriscoll 
Will this be fixed in 4.0.10?
